plugin-api/src/main/java/org/sonar/api/issue/impact/Severity.java

```diff
@@ -19,8 +19,14 @@
  */
 package org.sonar.api.issue.impact;
 
+import org.sonar.api.Beta;
+
 public enum Severity {
+  @Beta
```

Comment:
I am not sure anyone honors this annotation, so feel free to drop it ;) Anyway I think it would be hard to revert such a change.

Reply:
I think the idea was to be able to change them because when Leo created the PR, we were still unsure about the naming of the two new severities. Now that the names have been finalized, I can for sure remove it.